triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -385,7 +397,7 @@ class Runner(
   @throws[RuntimeException]
   private def handleCommands(commands: Seq[Command]): (UUID, SubmitRequest) = {
     val commandUUID = UUID.randomUUID
-    pendingCommandIds += ((commandUUID, SeenMsgs.Neither))
+    discard(pendingCommandIds.addOne(commandUUID -> SeenMsgs.Neither))
```

Comment:
```suggestion
    pendingCommandIds.update(commandUUID, SeenMsgs.Neither)
```
* avoid building a tuple
* avoid discard